### PR TITLE
chore(preview): remove classic theme - FE-3119

### PR DESCRIPTION
**Pull request by DipperTheDan**
removal of classic story, refactor stories file to CSF, create an MDX file and remove redundant
documentation

fixes FE-3119

### Proposed behaviour

- Remove redundant documentation
- Create an MDX example for Content
- Refactor stories file to be CSF
- Remove `classic` story

### Current behaviour

- Component still has our old style documentation
- No MDX example for Content
- Stories file is currently in `storiesOf` format
- `classic` story exists

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### Additional context
N/A

### Testing instructions
- `git checkout preview_remove_classic_theme`
- `npm start` to start Storybook
- Navigate to Preview in storybook
- Example should be in MDX and have no `classic` themed example

### CODE REVIEW

File: cypress/features/regression/preview.feature [hunk 1]
```diff
@@ -3,17 +3,17 @@ Feature: Preview default component
 
   @positive
   Scenario: Enable loading checkbox for a Preview component
-    When I open default "Preview" component in noIFrame with "preview" json from "commonComponents" using "loading" object name
+    When I open basic "Preview Test" component in noIFrame with "preview" json from "commonComponents" using "loading" object name
```
Comment (DlgSHi) [inline cypress/features/regression/preview.feature:6]: this TC (don't forget about `.json` file) could be deleted - it is covered by `chromatic`

File: cypress/features/regression/preview.feature [hunk 2]
```diff
@@ -3,17 +3,17 @@ Feature: Preview default component
...
     Then Preview component is loading
 
   @positive
   Scenario: Disable loading checkbox for a Preview component
-    When I open default "Preview" component in noIFrame with "preview" json from "commonComponents" using "loadingFalse" object name
+    When I open basic "Preview Test" component in noIFrame with "preview" json from "commonComponents" using "loadingFalse" object name
```
Comment (DlgSHi) [inline cypress/features/regression/preview.feature:11]: this TC (don't forget about `.json` file) could be deleted - it is covered by `chromatic`

File: cypress/features/regression/preview.feature [hunk 3]
```diff
@@ -22,7 +22,7 @@ Feature: Preview default component
 
   @positive
   Scenario Outline: Set width to <width>
-    When I open default "Preview" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
+    When I open basic "Preview Test" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
```
Comment (DlgSHi) [inline cypress/features/regression/preview.feature:25]: this TC (don't forget about `.json` file) could be deleted - it is covered by `chromatic`

File: cypress/features/regression/preview.feature [hunk 4]
```diff
@@ -32,7 +32,7 @@ Feature: Preview default component
 
   @positive
   Scenario Outline: Set height to <height>
-    When I open default "Preview" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
+    When I open basic "Preview Test" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
```
Comment (DlgSHi) [inline cypress/features/regression/preview.feature:35]: this TC (don't forget about `.json` file) could be deleted - it is covered by `chromatic`

File: cypress/features/regression/preview.feature [hunk 5]
```diff
@@ -42,7 +42,7 @@ Feature: Preview default component
 
   @positive
   Scenario Outline: Set lines to <lines>
-    When I open default "Preview" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
+    When I open basic "Preview Test" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
```
Comment (DlgSHi) [inline cypress/features/regression/preview.feature:45]: this TC (don't forget about `.json` file) could be deleted - it is covered by `chromatic`

**Review by mkrds — DISMISSED** [review]

**Comment by edleeks87** [comment]
Happy to approve once Igor's comments are addressed

**Review by DlgSHi — DISMISSED** [review]

**Review by mkrds — DISMISSED** [review]

File: src/components/preview/preview.component.js [hunk 6]
```diff
@@ -57,4 +40,21 @@ function createPreview(allProps, index) {
   );
 }
 
+Preview.propTypes = {
+  /** Children content to render in the component. */
+  children: PropTypes.node,
+  /** Classes to be applied to the component. */
```
Comment (dawidzarzycki) [inline src/components/preview/preview.component.js:46]: is this description ok about the classes?

**Review by dawidzarzycki — DISMISSED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by mkrds — APPROVED** [review]

File: cypress/features/regression/accessibility/accessibilityCommon.feature [hunk 7]
```diff
@@ -71,6 +70,7 @@ Scenario Outline: Component <component> basic story
     | step-sequence |
     | icon          |
     | button toggle |
+    | preview       |
```
Comment (KatarzynaQA) [inline cypress/features/regression/accessibility/accessibilityCommon.feature:73]: It's a wrong TC for `preview` component. We don't have just `basic` story. It's inside of `test` folder. So it should be a new TC like
 ```  
@accessibility
  Scenario Outline: Component <component> test basic page
    When I open "<component> Test" component page "basic" in no iframe
    Then "<component>" component has no accessibility violations
    Examples:
      | component        |
      | preview          |

```

File: cypress/features/regression/preview.feature [hunk 8]
```diff
@@ -1,58 +1,18 @@
...
-    When I open default "Preview" component in noIFrame with "preview" json from "commonComponents" using "<nameOfObject>" object name
-    Then Preview has <lines> lines
-    Examples:
-      | lines | nameOfObject |
-      | 0     | lines0       |
-      | 1     | lines1       |
-      | 100   | lines100     |
-
   @positive
   Scenario Outline: Set width to out of scope to <width>
```
Comment (KatarzynaQA) [inline cypress/features/regression/preview.feature:14]: I think we can remove this TC

File: cypress/features/regression/preview.feature [hunk 9]
```diff
@@ -61,7 +21,7 @@ Feature: Preview default component
 
   @positive
   Scenario Outline: Set height to out of scope to <height>
```
Comment (KatarzynaQA) [inline cypress/features/regression/preview.feature:23]: I think we can remove this TC

File: cypress/features/regression/preview.feature [hunk 10]
```diff
@@ -70,7 +30,7 @@ Feature: Preview default component
 
   @positive
   Scenario Outline: Set lines to out of scope to <lines>
```
Comment (KatarzynaQA) [inline cypress/features/regression/preview.feature:32]: I think we can remove this TC

**Review by KatarzynaQA — DISMISSED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by mkrds — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by KatarzynaQA — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by mkrds — APPROVED** [review]

**Pull request merged**